lib/webdriveragent.js

```diff
@@ -46,123 +48,210 @@ class WebDriverAgent {
       log.info(`Using default bootstrap: ${BOOTSTRAP_PATH}`);
     }
 
-    this.sim = args.sim;
+    this.device = args.device;
     this.platformVersion = args.platformVersion;
     this.host = args.host;
     this.agentPath = args.agentPath || AGENT_PATH;
     this.bootstrapPath = args.bootstrapPath || BOOTSTRAP_PATH;
   }
 
-  async launch (sessionId) {
+  async launch (sessionId, realDevice = false) {
     return new B(async (resolve, reject) => {
```

Comment:
I think it's worth cleaning up as part of this PR, even though it was pre-existing